Source/Tech Tree/Parts Browser/data/Procedural_Parts.json

```diff
@@ -48,6 +48,27 @@
             "NoResourceCostMult"
         ]
     },
+    {
+        "name": "proceduralBattery",
+        "title": "Procedural Battery (Deprecated)",
+        "description": "DO NOT USE THIS. Use Service Modules instead",
+        "mod": "Procedural Parts",
+        "cost": "999999",
```

Comment:
Not sure that's wise. The combination of title, description and non-RP0 should already be enough.

Reply:
I suppose I don't need to add any extra motivation